algorithms/unit_tests/BenchmarkStdAlgorithmsForEach.cpp

```diff
@@ -0,0 +1,67 @@
+//@HEADER
```

Comment:
FIXME: move this to a separate directory (`perf_tests`) and adjust CMakeLists.txt accordingly